src/System.Private.CoreLib/src/System/String.CoreCLR.cs

```diff
@@ -64,9 +64,6 @@ public partial class String
         // and not a - or '
         [MethodImplAttribute(MethodImplOptions.InternalCall)]
         internal extern bool IsFastSort();
```

Comment:
IsFastSort internal cal should be removed as well, shouldn't it?
Also I guess we can now remove that code to keep the string state, e.g. remove `InternalCheckHighChars` etc

Reply:
`IsFastSort` is still used in few places. Cleaning those up can be separate PR.